Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -3445,51 +3446,126 @@ def panorama_delete_url_filter_command(url_filter_name: str):
 
 ''' Security Rules Managing '''
 
+def xml_get(elem: dict, key: Any, subkeys = ('member', '#text', '@name')):
```

Comment:
we have dict_safe_get in CSP, lets use instead

Reply:
although i found some other places where i can use that

Reply:
it does work recursively, 
for example

    `alert_csv_id = dict_safe_get(alert, ['alert_data', 'csv', 'id'])`
check in the code how it is used

Reply:
I should have been more clear. heres an example:

in one rule you have:
```
"destination-hip": {
                        "@admin": "demistoadmin",
                        "@dirtyId": "12",
                        "@time": "2023/05/30 03:17:35",
                        "member": {
                            "@admin": "demistoadmin",
                            "@dirtyId": "12",
                            "@time": "2023/05/30 03:17:35",
                            "#text": "any",
                        },
                    }
```

and in another:

`"destination-hip": {"member": "any"}`

i need to get the value "any" regardless of how many keys deep it is
this function will dig until it gets a value and not a dict